iceoryx_binding_c/test/moduletests/test_notification_info.cpp

```diff
@@ -91,6 +91,10 @@ class iox_notification_info_test : public Test
         return m_memoryManager.getChunk(chunkSettings);
     }
 
+    static void triggerCallback(iox_sub_t sub)
+    {
+        (void)sub;
```

Comment:
Please use [`DISCARD_RESULT`](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_hoofs/include/iceoryx_hoofs/cxx/attributes.hpp#L31) here.

Reply:
`DISCARD_RESULT` is used to suppress compiler warnings for unused return values and not unused variables.

Reply:
What about `IOX_MAYBE_UNUSED`? :smirk:

Reply:
Correct :+1: